airbyte-integrations/connectors/source-railz/.dockerignore

```diff
@@ -0,0 +1,6 @@
+*
+!Dockerfile
+!main.py
+!source_railz
+!setup.py
+!secrets
```

Comment:
we should not pass the secrets dir in the docker image build. Could you please remove this line? 
```suggestion
```